docs/source/user_guide/installation/index.rst

```diff
@@ -4,42 +4,30 @@ Installation
 PyBaMM is available on GNU/Linux, MacOS and Windows.
 It can be installed using ``pip`` or ``conda``, or from source.
 
-.. tab:: GNU/Linux and Windows
+.. tab:: pip
 
-   .. tab:: pip
+   PyBaMM can be installed via pip from `PyPI <https://pypi.org/project/pybamm>`__.
 
-      PyBaMM can be installed via pip from `PyPI <https://pypi.org/project/pybamm>`__.
+   .. code:: bash
 
-      .. code:: bash
+      pip install pybamm
 
-         pip install pybamm
+.. tab:: conda
 
-   .. tab:: conda
+   PyBaMM is part of the `Anaconda <https://docs.continuum.io/anaconda/>`_ distribution and is available as a conda package through the conda-forge channel.
```

Comment:
(pedantic) let's correct this, since it has been incorrect for some time now. PyBaMM being a part of the Anaconda distribution means that it is a part of the `defaults` channel (which it isn't).
```suggestion
   PyBaMM is available as a ``conda`` package through the conda-forge channel.
```